lib/InputTest.ts

```diff
@@ -194,7 +250,7 @@ export class InputTest extends EventEmitter {
    * override the result from determining whether audio is silent from the collected volume levels.
    */
   stop(pass: boolean = true): InputTest.Report | undefined {
-    if (this._endTime) {
+    if (typeof this._endTime === 'number') {
```

Comment:
What else can this be?

Reply:
I changed this one because unit tests will return `Date.now() === 0` sometimes, so `this._endTime` is set to `0`.